README.md

```diff
@@ -75,7 +75,7 @@ installation. NOTE: We do not currently support python 3.11, please use an older
     - `libreoffice` (MS Office docs)
 - If you are parsing PDFs, run the following to install the `detectron2` model, which
   `unstructured` uses for layout detection:
-    - `pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@e2ce8dc#egg=detectron2"`
+    - make install-detectron2
```

Comment:
ahh, like funneling them to the make command and much cleaner than having that commit-specific url duped.